src/renderers/webgpu/utils/WebGPUBindingUtils.js

```diff
@@ -373,8 +373,29 @@ class WebGPUBindingUtils {
 
 					const usage = GPUBufferUsage.UNIFORM | GPUBufferUsage.COPY_DST;
 
+					const visibilities = [];
+					if ( binding.visibility & 1 ) {
```

Comment:
Instead of using 1,2 and 4 can you please use the real constants `GPUShaderStage.VERTEX`, `GPUShaderStage.FRAGMENT` and `GPUShaderStage.COMPUTE`? That makes the code more readable and robust.